playgame.py

```diff
@@ -59,17 +65,20 @@ def run_one_game(args):
 def prep_logger(log_dir, verbose, log_stderr, count):
     formatter = logging.Formatter('[%(asctime)s] %(levelname)8s --- %(message)s ' +
                                   '(%(filename)s:%(lineno)s)',datefmt='%Y-%m-%d %H:%M:%S')
+    
     if verbose:
         logger.setLevel(logging.DEBUG)
     else:
         logger.setLevel(logging.INFO)
     if log_dir:
-        fh = RotatingFileHandler('results/{dir}'.format(dir = log_dir), mode = 'w', backupCount = count)
+        ensure_path(log_dir)
+        fh = RotatingFileHandler('{dir}'.format(dir = log_dir), mode = 'w', backupCount = count)
```

Comment:
We can drop the string formatting now that we are just logging to log_dir
```suggestion
        fh = RotatingFileHandler('log_dir', mode = 'w', backupCount = count)
```